src/stratis_cli/_actions/_logical.py

```diff
@@ -54,10 +54,15 @@ def create_volumes(namespace):
             .search(managed_objects)
         )
 
-        (_, rc, message) = Pool.Methods.CreateFilesystems(
+        ((changed, _), rc, message) = Pool.Methods.CreateFilesystems(
             get_object(pool_object_path), {"specs": namespace.fs_name}
         )
 
+        if not changed:
```

Comment:
I _think_ the order of this code block and the next one should be reversed, i.e., check the value of ```rc``` first, and _then_ check ```changed```.